docs/README.md

```diff
@@ -23,7 +23,7 @@ _So you wanna build the site?_
 
 ## License
 
-:copyright: 2016-2018 [JS Foundation](https://js.foundation) and contributors.
+:copyright: 2016-2018 [OpenJS Foundation](https://openjsf.org/) and contributors.
```

Comment:
Isn't it necessary to change the year from 2018 to 2019, too? :)

Reply:
Nice catch! I reflected that too.